developer-guides/rest-api/channels/online/README.md

```diff
@@ -0,0 +1,41 @@
+# Channel Online
+
+List all online users of a channel.
+
+| URL | Requires Auth | HTTP Method |
+| :--- | :--- | :--- |
+| `/api/v1/channels.online` | `yes` | `GET` |
+
+## Payload
+
+| Argument | Example | Required | Description |
+| :--- | :--- | :--- | :--- |
+| `roomId` | `ByehQjC44FwMeiLbX` | Required | The channel's id |
+
+## Example Call
+
+```bash
+curl -H "X-Auth-Token: 9HqLlyZOugoStsXCUfD_0YdwnNnunAJF8V47U3QHXSq" \
+     -H "X-User-Id: aobEdbYhXfu5hkeqG" \
+     http://localhost:3000/api/v1/channels.online?roomId=74nd3hTbYrZbiTDp6
```

Comment:
@thayannevls this endpoint does not support `roomId` as query parameter, [as you can see here](https://github.com/RocketChat/Rocket.Chat/blob/develop/packages/rocketchat-api/server/v1/channels.js#L610), the endpoint wait for query parameter [like this](https://rocket.chat/docs/developer-guides/rest-api/query-and-fields-info/#query-example). You can filter by channel's id, filtering by `{"_id": "channel-id"}` otherwise the endpoint will retrieve the list of online users of all channels.

Reply:
I fixed it and added more info about the query parameter following examples from the others documentations.